storage/remote/otlptranslator/prometheusremotewrite/helper.go

```diff
@@ -540,6 +549,18 @@ func (c *PrometheusConverter) addTimeSeriesIfNeeded(lbls []prompb.Label, startTi
 	}
 }
 
+// handleStartTime adds a zero sample 1 millisecond before ts iff startTs == ts.
+func (c *PrometheusConverter) handleStartTime(startTs, ts int64, value float64, labels []prompb.Label, settings Settings) {
+	if !settings.EnableCreatedTimestampZeroIngestion {
+		return
+	}
```

Comment:
Do you have any special reason to add the whole `Settings` object as an argument and just use one of its fields? Would it be a problem to just use `EnableCreatedTimestampZeroIngestion` as argument?

Reply:
I think it's really just a matter of style, whether to pass the whole `Settings` object or individual fields. A reason to do it in this code base is to be consistent, since `Settings` is already passed around. The reason why I usually prefer this, is in case more settings get used. Then you won't have to rewrite everywhere.